gulpfile.js

```diff
@@ -249,13 +249,6 @@ gulp.task(
   }),
 );
 
-gulp.task(
-  'check-dependencies',
-  gulp.series(function() {
-    return gulp.src('package.json').pipe(gulpCheckDependencies());
```

Comment:
This is either a mistake according to confluence or you forgot to remove the import of `gulpCheckDependencies` as well.